docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+version: '3.2'
+
+services:
+  factomd:
+    network_mode: host
+    container_name: factomd_container
+    image: factominc/factomd:latest
+    entrypoint: /go/bin/factomd
+    command: '--count=5 --net=alot+ --blktime=15 --faulttimeout=20000 --enablenet=false --network=LOCAL --startdelay=2'
```

Comment:
Any reason to spawn 5 simulated nodes?
Also I don't remember what does `--net` and `--enablenet` do, what are they and why are they necessary here?

Reply:
No particular reason, I was following this script https://github.com/FactomProject/factomd/blob/master/scripts/g25.sh

Please suggest what to change, I am not 100% sure if that is the best settings for local testing =)

Reply:
Ok then I would keep it as simple as:
`--network=LOCAL --blktime=15`

Reply:
Changed